src/main/java/org/beanpod/switchboard/controller/DecoderController.java

```diff
@@ -72,6 +72,9 @@
 
   @PostMapping
   public ResponseEntity<DecoderDto> createDecoder(@RequestBody @Valid DecoderDto decoderDto) {
+    if (decoderDto.getInput().isEmpty()) {
```

Comment:
Why not trying to use @NotEmpty annotation in the DecoderDto class?

Reply:
doesnt work

Reply:
All javax validations do not work on the set for some reason: notempty & size=1